FAQ.md

```diff
@@ -14,7 +14,7 @@ In addition, APISIX has dynamic routing and hot loading of plug-ins, which is es
 
 ## What's the performance of APISIX?
 
-One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md。
+One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](https://github.com/iresty/apisix/blob/master/doc/benchmark-cn.md)
```

Comment:
Please use this link: https://github.com/iresty/apisix/blob/master/doc/benchmark.md